oshc/main/templates/base.html

```diff
@@ -29,12 +29,12 @@
             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Sessions <span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
               <li><a href="https://www.youtube.com/channel/UC1_IAby-9me3iICtxuiDL5Q" target="_blank">View Session</a></li>
-              <li><a href="{% url 'request_session' %}" target="_blank">Request a Session</a></li>
+              <li><a href="{% url 'request_session' %}" target="_self">Request a Session</a></li>
             </ul>
           </li>
           <li><a href="https://github.com/OpenSourceHelpCommunity" target="_blank">Resources</a></li>
-          <li><a href="{% url 'contests' %}" target="_blank">Contests</a></li>
-          <li><a href="{% url 'journey' %}" target="_blank">Journey</a></li>
+          <li><a href="{% url 'contests' %}" target="_self">Contests</a></li>
```

Comment:
👋 it defaults to `_self`, I think. Do you know about any advantages of using `_self` in target?

Reply:
I am not aware of any specific advantages as far as function. I do develop a few sites hosted on my raspberry pi (which is why I just recently started out with github these past few days), and I like having functions available in case I want to do something with them at a later time.

Reply:
@tapasweni-pathak I tried to explore about the same, what I concluded was that there are no such advantages it's just that the default behaviour is expected to open the linked document in the same frame.